helm-gcs-publish/action.yml

```diff
@@ -1,6 +1,9 @@
 name: 'Helm GCS Publish'
 description: 'Published helm charts to GCS'
 inputs:
+  role-name: 
```

Comment:
Why is the neded as an argument to the action? The publish action's role is always... publish, never validate. we can hardcode that on line 21 and making this simpler to use.

Reply:
done